test/jdk/java/net/httpclient/SpecialHeadersTest.java

```diff
@@ -30,6 +30,7 @@
  * @build jdk.httpclient.test.lib.common.HttpServerAdapters
  *        jdk.httpclient.test.lib.http2.Http2TestServer
  *        jdk.test.lib.net.SimpleSSLContext
+ * @requests (vm.compMode != "Xcomp")
```

Comment:
I didn't notice this change while reviewing - it is causing trouble in the CI.

Reply:
Is PR #12399 related?

Reply:
yes - is that RMI test in https://github.com/openjdk/jdk/pull/12399 using an HttpServer?

Reply:
It seems that here `@requests` should have been `@requires` ; proposed fix https://github.com/openjdk/jdk/pull/12409

Reply:
I doubt that what is proposed in https://github.com/openjdk/jdk/pull/12399 is the right fix for the issue then. I commented there. Still looking for a reviewer for https://github.com/openjdk/jdk/pull/12409 BTW ;-)

Reply:
I'm sorry for my low-level typo ;-(